docs/reference/configuration/messaging.md

```diff
@@ -125,7 +125,7 @@ Im folgenden werden nun alle erforderlichen Parameter erklärt.
 - type: telegram
   token: # bot id
   chats:
-  - # list of chat ids
+  - # list of chat ids, die ChatIds müssen mit einem weiteren -Zeichen starten
```

Comment:
Vielleicht:

```
  - # Liste von Chad IDs. Jeder Eintrag benötigt laut YAML eine `-` Zeichen am Anfang und muss in einer eigenen Zeile sein.
```

Reply:
Alternativ könnte man auch die Kurzschreibform von Listen verwenden, wenn das einfacher zu erklären ist. Die sieht so aus:

```
  chats: [ "<chat id>", "<chat id>", ...] # Der Text `<chat id>` muss durch die entsprechende ID ersetzt werden. Bei mehreren werden diese durch `,` getrennt
```

oder so etwas in der Art.

Reply:
Hallo,
am besten gefällt mir dieser Vorschlag von dir:
"# Liste von Cha**t** IDs. Jeder Eintrag benötigt laut YAML ein `-` Zeichen am Anfang und muss in einer eigenen Zeile sein."

Die Listenform ist viel zu kompliziert, ich hab als Anfänger ewig gebraucht bis ich das mit dem extra - Zeichen kapiert habe. Die meisten haben auch nur eine Chat ID